Include/internal/pycore_bitutils.h

```diff
@@ -131,8 +131,40 @@ _Py_popcount32(uint32_t x)
 }
 
 
+// Return the index of the most significant 1 bit in 'x'. This is the smallest
+// integer k such that x < 2**k. Equivalent to floor(log2(x)) + 1 for x != 0.
+static inline int
+_Py_bit_length(unsigned long x) {
+    if (x == 0) {
+        return 0;
+    }
+#if (defined(__clang__) || (defined(__GNUC__) && (((__GNUC__ == 3) && (__GNUC_MINOR__ >= 4)) || (__GNUC__ >= 4))))
```

Comment:
Sorry, I don't think that we should bother with supporting GCC older than 3.4. Just test for `__GNUC__`, and instead add the comment:

```
// __builtin_clzl() is available since GCC 3.4.
```

**If** someone reports a build failure, we can consider supporting such old GCC version :-)

Reply:
I figured the actual test is not much more verbose than the comment, but good point: We probably want to actually know this, instead of silently using the fallback. Done.